modules/ROOT/pages/managing-users.adoc

```diff
@@ -50,6 +50,16 @@ Public key provided by the identity provider, which is used to sign the SAML ass
 +
 An arbitrary string value that identifies your Anypoint Platform organization. The typical value for this string is `<organizationDomain>.anypoint.mulesoft.com`.
 +
+* *Single Sign-On Initiation*
++
+Specify whether SSO can be initiated by the Anypoint Platform, your identity provider (e.g., Okta), or both. 
+
```

Comment:
If the following are subbullet list items, they should be formatted using double astrisks